src/web/components/ParticipantRequests/ParticipantRequestItem.scss

```diff
@@ -8,6 +8,13 @@
     margin-right: 8px;
   }
 
+  .participant-request-api-role-label {
```

Comment:
It feels a bit odd that the ApprovedParticipantItem is using styling from ParticipantRequestItem.scss. It does make sense that they share some styling, but maybe there is a more clear way to can do this. Few options:
-  Have a common scss file, something like `ManageParticipantItem.scss` which is shared by approved participants and participant requests.
- Completely separate the scss files so that ParticipantRequestItem is the only user of ParticipantRequestItem.scss - and ApprovedParticipantItem is the only user of ApprovedParticipantItem.
- Something else?

I could be wrong here though - what do you think?

Reply:
I think that's a fair observation. This set of components has a slightly strange naming all around since it contains the approved participants table, the requested participants and the form for that. My opinion would be to rename the whole folder they're in to be more along the lines of ```ManagementParticipant``` and then rename the two scss files to be along the lines of ```ManagementParticipantTable.scss``` and  ```ManagementParticipantItem.scss``` then slightly rename the classes inside to reflect this.

Reply:
That name is also what the stories are actually under on storybook.
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/152250444/c5843c1b-e0d1-4d7d-bcbd-41e4ec68ed49)

Reply:
Something like that sounds good to me, as long as we can easily tell the difference between the participant requests and approved participants (as well as stuff that is shared!).

Reply:
Have done a bunch of refactoring to it. Sorry for the mass of changes.

See screenshots below of unchanged storyboard.
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/152250444/8dc1a614-0172-46e8-b285-c31e37428878)
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/152250444/b7062adb-6196-4f64-a426-132625bb073a)

Reply:
The font weight of the API Roles looks a bit too strong now (i.e. it looks bold) otherwise looks good visually! Still reading the actual code

Reply:
I think it is inheriting a font-weight of 600 from the tooltip, if that helps.

Reply:
Yeah it was. I think that this is the only spot the trigger as text was given so just adjusted its formatting.